Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -93,9 +93,12 @@ void ISerializationCallbackReceiver.OnAfterDeserialize()
             // *Note This will only take effect once the Mixed Reality Toolkit has a custom editor for the MixedRealityConfigurationProfile
 
             ActiveManagers.Clear();
-            for (int i = 0; i < initialManagers.Length; i++)
+            if (initialManagers != null && initialManagers.Length > 0)
```

Comment:
` initialManagers?.Length > 0`

Reply:
Actually `for (int i = 0; i < initialManagers?.Length; i++)`

Reply:
Fair enough